src/BuildingRegistry.Api.BackOffice.Abstractions/Validation/ChangeBuildingUnitFunction.cs

```diff
@@ -0,0 +1,26 @@
+namespace BuildingRegistry.Api.BackOffice.Abstractions.Validation
+{
+    using TicketingService.Abstractions;
+
+    public static partial class ValidationErrors
+    {
+        public static class ChangeBuildingUnitFunction
+        {
+            public static class BuildingInvalidStatus
+            {
+                public const string Code = "GebouwStatusNietInGeplandInAanbouwOfGerealiseerd";
+                public const string Message = "Deze actie is enkel toegestaan binnen een gepland, inAanbouw of gerealiseerd gebouw.";
+
+                public static TicketError ToTicketError() => new(Message, Code);
```

Comment:
Nice

Reply:
Yeah, but every sub class needs this static method. Maybe we should look for a generic way of doing this. But this is a start.

Reply:
Yes we need a base class with Code, Message props and a static ToTicketError method, something that doesn't break the static chaining of classes